xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-api/src/main/java/org/xwiki/refactoring/script/RefactoringScriptService.java

```diff
@@ -410,7 +251,22 @@ public Job convertToTerminalDocument(DocumentReference documentReference)
      * @param request the copy request to perform
      * @return the job that has been scheduled and that can be used to monitor the progress of the operation,
      *         {@code null} in case of failure
+     * @since 10.11RC1
      */
+    public Job copy(CopyRequest request)
+    {
+        return execute(RefactoringJobs.COPY, request);
+    }
+
+    /**
+     * Schedules an asynchronous job to perform the given copy request.
+     *
+     * @param request the copy request to perform
+     * @return the job that has been scheduled and that can be used to monitor the progress of the operation,
+     *         {@code null} in case of failure
+     * @deprecated Use {@link #copy(CopyRequest)} instead.
+     */
+    @Deprecated
     public Job copy(MoveRequest request)
```

Comment:
Shouldn't this be moved to legacy too ?

Reply:
yep. Sorry it's still a WIP I forgot to update the title.